src/support/global.c

```diff
@@ -40,6 +40,72 @@ __wt_endian_check(void)
 	return (EINVAL);
 }
 
+/*
+ * __global_calibrate_ticks --
+ *	Calibrate a ratio from rdtsc ticks to nanoseconds.
+ */
+static void
+__global_calibrate_ticks(void)
+{
+#if defined (__i386) || defined (__amd64)
+	struct timespec start, stop;
+	double ratio;
+	uint64_t diff_nsec, diff_tsc, min_nsec, min_tsc;
+	uint64_t tries, tsc_start, tsc_stop;
+	volatile uint64_t i;
+
+	min_nsec = min_tsc = 0;
+	/*
+	 * Run this calibration loop a few times to make sure we get a
+	 * reading that does not have a potential scheduling shift in it.
+	 * The inner loop is CPU intensive but a scheduling change in the
+	 * middle could throw off calculations. Take the minimum amount
+	 * of time and compute the ratio.
+	 */
+	__wt_process.use_epochtime = false;
```

Comment:
I don't think this value is ever read, my bet is clang will complain.

Reply:
It is used in `wt_rdtsc` to determine if we should always use `wt_epoch` so it needs to be false at this point. Probably a comment would be good. This is where a flag would be better because a `USE_EPOCH_TIME` flag would not yet be set and the code checks if it is.